crates/bevy_ecs/src/storage/sparse_set.rs

```diff
@@ -113,6 +130,8 @@ pub struct ComponentSparseSet {
 }
 
 impl ComponentSparseSet {
```

Comment:
What makes it "sparse" if you catch my meaning? How does it work? I think that's what I really want to know.

Reply:
SparseSets are sparse because the ID space they occupy do not always have an element. An entity doesn't *have* to have a `Transform` component, so if it were stored in a `ComponentSparseSet`, it wouldn't store a value for every entity.